export default async ({ store }) => {
  try {
    // await store.dispatch('auth/ME')
  } catch (err) {
    console.log('me err: ', err)
  }
}
